### PR TITLE
VC module

### PR DESCRIPTION
**Pull request by DaniBodor**
changes to version control module

### CODE REVIEW

File: modules/version-control/slides-version-control.md [hunk 1]
```diff
@@ -72,9 +72,58 @@ The changes are stored along with some useful metadata. A set of changes, along
...
+This line contains typos.
+=======
+This line contains a typo.
+>>>>>>> contributor2
+```
+<!-- .element: class="fragment" data-fragment-index="2" -->
+
+note:
+
+If changes are made to the same section (usually the same or consectutive line(s) of text) of a document a **conflict** aruises. Changes cannot be automatically merged, as the interpreter does not know which version or which combination to use. Human intervention is required and can involve rolling back a change, finding common ground between changes, etc.
```
Comment (JaroCamphuijsen) [inline modules/version-control/slides-version-control.md:103]:
```suggestion
If changes are made to the same section (usually the same or consecutive line(s) of text) of a document a **conflict** arises. Changes cannot be automatically merged, as the interpreter does not know which version or which combination to use. Human intervention is required and can involve rolling back a change, finding common ground between changes, etc.
```

File: modules/version-control/slides-version-control.md [hunk 2]
```diff
@@ -72,9 +72,58 @@ The changes are stored along with some useful metadata. A set of changes, along
...
+
+- Communicate:
+  - discuss who is working on what
+  - agree on common standards such as formatting, (conflicting) style guides, linting rules, etc.
+- Frequent merging:
+  - Merge (small) changes back to the central repository often
+  - Pull changes to your local repository regularly
+- Organize changes:
+  - use "feature branches" rather than making changes to the main document directly 
+  - avoid "scope creep" and instead create separate indepedent branches for isolated topics
```
Comment (JaroCamphuijsen) [inline modules/version-control/slides-version-control.md:120]:
```suggestion
  - avoid "scope creep" and instead create separate independent branches for isolated topics
```

File: modules/version-control/version_control_exercise.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: What to include in the repository
+type: exercise
+order: 3
+---
+
+## Exercise: Which of the following do you think should be part of a code repository (5-10 mins)
```
Comment (JaroCamphuijsen) [inline modules/version-control/version_control_exercise.md:7]: Do we need extra information here about the format of this exercise? E.g. 
```suggestion
## Exercise: Which of the following do you think should be part of a code repository (5-10 mins)

Discuss with your neighbor which of the following should be part of the code repository and why it should or should not.
```

Comment (JaroCamphuijsen) [inline modules/version-control/version_control_exercise.md:7]: Do we need extra information here about the format of this exercise? E.g. 
```suggestion
## Exercise: Which of the following do you think should be part of a code repository (5-10 mins)

Discuss with your neighbor which of the following should be part of the code repository and why it should or should not.
```

File: modules/version-control/version_control_exercise.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+- installation instructions
```
Comment (JaroCamphuijsen) [inline modules/version-control/version_control_exercise.md:9]:
```suggestion
- installation instructions (i.e. detailed instructions on how to install the software)
```

File: modules/version-control/version_control_exercise.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+- user documentation
```
Comment (JaroCamphuijsen) [inline modules/version-control/version_control_exercise.md:10]:
```suggestion
- user documentation (i.e. information for the users of the software about its functionality and features)
```

**Review by JaroCamphuijsen — APPROVED** [review]
I like your additions @DaniBodor. I left some minor comments.

**Comment by DaniBodor** [comment]
> I like your additions @DaniBodor. I left some mino suggestions.

Good suggestions, thanks!

**Pull request merged**